### ISSUE TITLE
Implement SA1136: Enum values should be on separate lines

### ISSUE THREAD

**Issue by NikolayIT** [DotNetAnalyzers/StyleCopAnalyzers#1854]
**Category:** Readability
**Title:** Enum values should be on separate lines

Code that will trigger it:

```cs
public enum SomeEnumType
{
    FirstValue, SecondValue, ThirdValue
}
```

Corrected code:
```cs
public enum SomeEnumType
{
    FirstValue,
    SecondValue,
    ThirdValue
}
```

**Comment by nbarbettini** [comment]
:+1: on this.

**Comment by sharwell** [comment]
:+1:

**Comment by sharwell** [comment]
:memo: I would prefer a single rule that applied to elements (members of a type or namespace) in general.

**Comment by vweijsters** [comment]
:+1:

**Comment by Maxwe11** [comment]
:+1:

**Comment by GregReddick** [comment]
:+1:

**Comment by vweijsters** [comment]
Grabbing this

**Comment by vweijsters** [comment]
This will be SA1136, there already is an issue for SA1135 (#1762)
